ganttproject/src/net/sourceforge/ganttproject/UIFacadeImpl.java

```diff
@@ -305,9 +306,17 @@ public void showPopupMenu(Component invoker, Action[] actions, int x, int y) {
   @Override
   public void showPopupMenu(Component invoker, Collection<Action> actions, int x, int y) {
     JPopupMenu menu = new JPopupMenu();
+
+    // TODO: refactor code so that submenus could be added in more generic way
+    String assignTo = GanttLanguage.getInstance().getText("assignment.toResource");
```

Comment:
I suggest using existing "assignments" string. It looks better fit, because the submenu deals with assignments (in plural) and allows not only for "assign" action but for assignment removal as well.

Reply:
OK, it's reasonable. Thanks for the explanation.